docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -109,9 +109,9 @@ all labeled and additional information will be included below.
     - This is location specific, but not age specific. Currently assume that there is no correlation of ANC with other factors. Engineers, you can pull these value straight from GBD, but expected values are as follows - Ethiopia: 75.7%, Nigeria: 74.3%, Pakistan: 90.8%
   * - 2
     - Ultrasound rate at ANC in baseline scenario
-    - 60.7% 
-    - `Linked ultrasound rate <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9201203/>`_ 
-    - Data value is a stand-in as we finalize. We would like this to be location specific, please code accordingly.
+    - Ethiopia: 60.7%, Nigeria: 58.7%, Pakistan: 61.4%
```

Comment:
Is this e.g. 60.7% of the 75.5 pregnant people who attended ANC (out of 100) in Ethiopia?  Or is it 60.7 pregnant people out of 100?

Reply:
I just double-checked, and all 3 sources are among pregnant people only (rather than all WBPRA). However, the studies in Ethiopia and Nigeria are specifically about pregnant women attending an ANC facility. Because we have data regarding ANC visits for all 3 locations, maybe we can adjust the India value to only include pregnant people attending an ANC? 
cc @NathanielBlairStahn

Reply:
Good to know! I think a single number for pregnant people attending ANC any number of times is good for now.